wikipedia_search_and_save/wikipedia_search_and_save.py

```diff
@@ -0,0 +1,63 @@
+import wikipedia
+from fpdf import FPDF
+
+
+class PDF(FPDF):
+    def chapter_title(self, label):
+        # Arial 12
+        self.set_font('Arial', '', 12)
+        # Background color
+        self.set_fill_color(200, 220, 255)
+        # Title
+        self.cell(0, 6, '%s' % (label), 0, 1, 'L', 1)
+        # Line break
+        self.ln(4)
+
+    def chapter_body(self, file_name):
+        # Read text file
+        with open(file_name, 'rb') as fh:
```

Comment:
Hey, is it possible to create a temp file if not pre-exist like in try-catch block

Reply:
Hey @pawangeek!
**I didn't quite understand well what you referred to in this comment. Please explain a little bit so that I can improve the code.** 
Here, I am doing the following tasks to convert to pdf. 
1. Opening a text file (creating if it doesn't already exists), say File_1
2. Writing the result of the wikipedia.summary in it.
3. Converting the text file into pdf.
4. Clearing the text in File_1.

I am using the same file (File_1) to store result for all the queries temporarily.

Reply:
Ohk, point 1 covers my point

Reply:
So, can I conclude this conversation is resolved?

Reply:
Yes, but some work needed for 2nd one